src/openrct2/peep/Peep.h

```diff
@@ -632,10 +634,14 @@ struct Peep : SpriteBase
     money16 PaidOnDrink;
     uint8_t RideTypesBeenOn[16];
     uint32_t ItemExtraFlags;
-    uint8_t Photo2RideRef;
-    uint8_t Photo3RideRef;
-    uint8_t Photo4RideRef;
-    uint8_t CurrentRide;
+    ride_id_t Photo2RideRef;
+    ride_id_t Photo3RideRef;
+    ride_id_t Photo4RideRef;
+    union
+    {
+        ride_id_t CurrentRide;
+        ParkEntranceIndex ChosenParkEntrance;
```

Comment:
I don't see any reason to make this into a union. Just have two separate fields. Or are you trying to simplify the import export code?

Reply:
The only reason I did this is because it was already like this. I think CS did it because `ChosenParkEntrance` is only used outside the park, while `CurrentRide` is only used in it.